requirements.txt

```diff
@@ -6,20 +6,20 @@ matplotlib
 xarray
 ipykernel
 h5netcdf
-torch
-pytorch-lightning
-perceiver_pytorch
+torch>=2.0
+pytorch-lightning>=2.0
 pytest
 pytest-cov
 flake8
 jedi
 tables
 tilemapbase
-neptune-client[pytorch-lightning]
+wandb
```

Comment:
for production, we might want to move some of these to extra requirments, but lets see